src/main/java/com/theelixrlabs/healthcare/constants/MessageConstants.java

```diff
@@ -19,5 +20,6 @@ public class MessageConstants {
     public static final String INVALID_UUID = "uuid.invalid";
     public static final String LAST_NAME_SHOULD_BE_MANDATORY = "{doctor.lastName.mandatory}";
     public static final String LAST_NAME_SHOULD_NOT_EMPTY = "doctor.lastName.should.not.be.empty";
```

Comment:
Rename LAST_NAME_SHOULD_NOT_EMPTY to LAST_NAME_SHOULD_NOT_BE_EMPTY

Reply:
changed to LAST_NAME_SHOULD_NOT_BE_EMPTY